packages/semver/src/executors/version/index.ts

```diff
@@ -227,6 +229,7 @@ function _normalizeOptions(options: VersionBuilderSchema) {
     changelogHeader: options.changelogHeader ?? defaultHeader,
     versionTagPrefix: options.tagPrefix ?? options.versionTagPrefix,
     commitMessageFormat: options.commitMessageFormat as string,
+    skipCommit: options.skipCommit || false,
```

Comment:
The default value should be placed in the `schema.json`, here you can type assert `options.skipCommit as boolean`.

Reply:
fixed